src/commands/start.command.ts

```diff
@@ -9,7 +9,8 @@ export const start = async (apps: AppDefinition[]) => {
 
   function startProcess(app: AppDefinition) {
     console.log("starting " + app.script);
-    const p = spawn("bash", ["-c", app.script]);
+    const NPM_PATH = execSync("npm bin").toString().trim();
```

Comment:
this should be optional or configurable as interpreter setting, npm can be default for example

Reply:
you mean to be independent of npm? (yarn or even non-node environment)

Reply:
yes but this can be done later :+1: